docs/content/latest/secure/authentication/password-authentication

```diff
@@ -0,0 +1,158 @@
+- The `ysql_hba_conf` flag above specifies rules that allow both MD5 and SCRAM-SHA-256 *existing* passwords to be used to connect to databases. 
+
+2. Start the YugabyteDB cluster.
+
+3. Open the YSQL shell (`ysqlsh`), specifying the `yugabyte` user and prompting for the password.
+
+```sh
+$ ./ysqlsh -U yugabyte -W
+```
+
+When prompted for the password, enter the `yugabyte` password (default is `yugabyte`). You should be able to log in and see a response like this:
+
+```
+ysqlsh (11.2-YB-2.2.2.0-b0)
+Type "help" for help.
+
+yugabyte=#
+```
+
+4. Change the password for `yugabyte` to a SCRAM-SHA-256 password.
+
+You can use either the ALTER ROLE statement or the `ysqlsh` `\password\` metacommand to change the password. The new password is encrypted using the SCRAM-SHA-256 hashing algorithm. In the following example, the `\password` metacommand is used to change the password.
+
+```sh
+yugabyte=# \password
+```
+
+You will be prompted twice for the new password and then returned to the YSQL shell prompt.
+
+```
+Enter new password:
+Enter it again:
+yugabyte=#
+```
+
+5. Stop the YugabyteDB cluster.
+
+6. Remove the MD5 rules from the `--ysql_hba_conf` flag.
+
+In the flagfile, the updated flag should appear like this:
+
+```sh
+--ysql_hba_conf=host all all 0.0.0.0/0 scram-sha-256,host all all ::0/0 scram-sha-256
+```
+
+7. Restart the YugabyteDB cluster.
+
+8. Open the YSQL shell and log in, specifying the `yugabyte` user and password prompt.
+
+```sh
+$ ./ysqlsh -U yugabyte -W
+```
+
+When prompted, the changed `yugabyte` user password should get you access. Any new users or roles that you create will be encrypted using SCRAM-SHA-256. Access to the host and databases is determined by the rules you specify in the YB-TServer `--ysql_hba_conf` configuration flag.
+
+## Migrate existing MD5 passwords to SCRAM-SHA-256
+
+When you [enable SCRAM-SHA-256 authentication](#enable-scram-sha-256-authentication) on an existing YugabyteDB cluster that has users and roles, with their MD5 passwords), you need to be aware that:
+
+- All new, or changed, passwords will be encrypted using the SCRAM-SHA-256 hashing algorithm.
+- All existing passwords were encrypted using the MD5 hashing algorithm.
+
+Because all existing passwords must be changed, you can manage the migration of these user and role passwords from MD5 to SCRAM-SHA-256 by maintaining rules in the `--ysql_hba_conf` setting to allow both MD5 passwords and SCRAM-SHA-256 passwords to work until all passwords have been migrated to SCRAM-SHA-256. For an example, see [Create a cluster that uses SCRAM-SHA-256 password authentication](Create-a-cluster-that-uses-scram-sha-256-password-authentication) above. If you follow a similar approach for an existing cluster, you can enhance your cluster security, track and migrate passwords, and then remove the much weaker MD5 rules after all passwords have been updated.
```

Comment:
```suggestion
Because all existing passwords must be changed, you can manage the migration of these user and role passwords from MD5 to SCRAM-SHA-256 by maintaining rules in the `--ysql_hba_conf` setting to allow both MD5 passwords and SCRAM-SHA-256 passwords to work until all passwords have been migrated to SCRAM-SHA-256. For an example, see [Create a cluster that uses SCRAM-SHA-256 password authentication](#Create-a-cluster-that-uses-scram-sha-256-password-authentication) above. If you follow a similar approach for an existing cluster, you can enhance your cluster security, track and migrate passwords, and then remove the much weaker MD5 rules after all passwords have been updated.
```
Add `#` on `(#Create-a-cluster-that-uses-scram-sha-256-password-authentication)` to fix the link